fg21sim/extragalactic/pointsource/base.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2016 Zhixian MA <zxma_sjtu@qq.com>
+# MIT license
+
+"""
+Simulation of point sources for 21cm signal detection
+
+Point sources types
+-----
+1. Star forming (SF) galaxies
+2. Star bursting (SB) galaxies
+3. Radio quiet AGN (RQ_AGN)
+4. Faranoff-Riley I (FRI)
+5. Faranoff-Riley II (FRII)
+
+References
+------------
+[1] Wilman R J, Miller L, Jarvis M J, et al.
+    A semi-empirical simulation of the extragalactic radio continuum sky for
+    next generation radio telescopes[J]. Monthly Notices of the Royal
+    Astronomical Society, 2008, 388(3):1335–1348.
+[2] Jelic et al.
+[3] Healpix, healpy.readthedocs.io
+"""
+# Import packages or modules
+import os
+# import healpy as hp  # healpy
+import numpy as np
+import time
+from pandas import DataFrame
+import astropy.units as au
+# Custom module
+import psparams
+
+# Defination of the base class
+class pointsource:
+    """
+    The basic class of point sources
+
+    Parameters
+    ----------
+    z: float
+        Redshift, z ~ U(0,20)
+    dA: au.Mpc
```

Comment:
Type should be ``float``; and describe its unit is ``au.Mpc`` in the following description.